tests/integration/features/access_control_replicated_backup.feature

```diff
@@ -45,14 +45,7 @@ Feature: Backup and restore functionality of replicated access control entities
     Then we got the following backups on clickhouse01
       | num | state   | data_count | link_count |
       | 0   | created | 1          | 0          |
-    When we execute command on clickhouse01
```

Comment:
Not needed anymore, because we don't copy entities from the ZK to `/var/lib/clickhouse/access`

Reply:
also, if we don't want to clean up access/tmp folders - should we change this step for checking tmp dir instead?